native/src/components/Header.tsx

```diff
@@ -151,12 +151,20 @@ const Header = (props: PropsType): ReactElement => {
   const renderItem = (
     title: string,
     show: 'never' | 'always',
-    onPress: () => void,
     accessibilityLabel: string,
```

Comment:
I think we could remove the accessibility label as it is always the same as the title. Furthermore we could just pass the key and call the `t` function here.

Reply:
We could also pass `show` as boolean and convert it to `never` or `always` here. Of course you don't have to do that, just if you want :)